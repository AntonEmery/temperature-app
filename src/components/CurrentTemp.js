import React from 'react';

function CurrentTemp({temp_c, temp_f}) {
  return (
    <div>
      <h3>Current Temperature</h3>
      <p>Fahrenheit: {temp_f}</p>
      <p>Celsius: {temp_c}</p>
    </div>
  )
}

export default CurrentTemp;