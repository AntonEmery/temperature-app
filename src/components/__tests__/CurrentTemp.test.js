import React from 'react';
import renderer from 'react-test-renderer';
import CurrentTemp from '../CurrentTemp';

describe('Current temp component', () => {
  it('Component renders given correct prop', () => {
    const currentTemp = 30;
    const tree = renderer.create(<CurrentTemp currentTemp={currentTemp} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
