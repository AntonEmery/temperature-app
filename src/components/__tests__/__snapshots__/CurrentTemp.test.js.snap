// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Daily temp component Current temp renders when temp is passed in as props 1`] = `
<div>
  <p>
    30
     degrees
  </p>
</div>
`;
